.github/workflows/conformance.yml

```diff
@@ -4,8 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request_target:
-    types: [labeled]
+  pull_request:
   workflow_dispatch:
```

Comment:
I don't think we necessarily *need* workflow dispatch anymore either?

Reply:
Nope, shouldn't be necessary. You can keep that around if you'd like to be able to manually trigger conformance tests, but IIRC we don't both with that in the `sigstore-python` tests either.